roles/kubernetes-apps/external_cloud_controller/openstack/templates/external-openstack-cloud-config.j2

```diff
@@ -57,7 +57,11 @@ internal-lb={{ external_openstack_lbaas_internal_lb }}
 {% endif %}
 {% if external_openstack_lbaas_provider is defined %}
 lb-provider={{ external_openstack_lbaas_provider }}
+{% if external_openstack_lbaas_use_octavia is defined %}
+use-octavia={{ external_openstack_lbaas_use_octavia }}
+{% else %}
```

Comment:
How about just
```
use-octavia={{ external_openstack_lbaas_use_octavia }}
```
without any condition with the default value `false` of external_openstack_lbaas_use_octavia to make it simple?

Reply:
u mean like that? 
```yaml
{% if external_openstack_lbaas_provider is defined %}
lb-provider={{ external_openstack_lbaas_provider }}
use-octavia={{ external_openstack_lbaas_use_octavia | default(False) }}
{% else %}
lb-provider=octavia
use-octavia=true
{% endif %}
```
I think this good idea. Should i commit this?